src/main/flight/pid.c

```diff
@@ -108,6 +108,7 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .dterm_notch_hz = 260,
         .dterm_notch_cutoff = 160,
         .dterm_filter_type = FILTER_PT1,
+        .dterm_filter_style = KD_FILTER_CLASSIC,
```

Comment:
Why are we defaulting to classing?

Reply:
the idea is to be less invasive as possible and to make sure we don't break anyone upgrading. If they find that this does a better job, we should absolutely default to the appropriate filtering style.

Reply:
NOSP is awesome !